code/modules/power/engines/singularity/singularity.dm

```diff
@@ -35,9 +35,14 @@
 	var/isnt_shutting_down = FALSE
 	/// Init list that has all the areas that we can possibly move to, to reduce processing impact
 	var/list/all_possible_areas = list()
+	/// Id for monitoring.
+	var/singulo_id = 1
+	/// Amount of singulos created during the round(note that this will include teslas as well)
+	var/static/global_singulo_id = 1
```

Comment:
Can you make this a glob instead of a static on the type?

Reply:
will do

Reply:
ddone